dll/opengl/glu32/src/libnurbs/internals/trimregion.h

```diff
@@ -46,6 +46,10 @@ class Backend;
 class TrimRegion {
 public:
 			TrimRegion();
+#ifdef __REACTOS__
+    /* FIXME: Inspect */
+    ~TrimRegion();
+#endif
```

Comment:
Why is this required?

Reply:
Unresolved reference at link time.

Reply:
Maybe because it's empty?

Reply:
This is really odd. It should get an implicit inline destructor. Something forcing a vtable entry? Hmm.